persist-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -44,7 +58,9 @@ OPTIONS
 BALLERINA COMMANDS
        The below is a list of available subcommands:
 
-       init            Initialize the package for persistence
+       add             Initialize the package for persistence. Used along with `bal build` command.
```

Comment:
```suggestion
       add             Initialize the package for persistence and integrate the client generation to the `bal build` command.
```

Reply:
Addressed the suggestions in [103a302](https://github.com/ballerina-platform/persist-tools/pull/336/commits/103a3022b5e3836c115a576dec082b7d2f2116c4)